plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -58,12 +59,15 @@ func getResourceRequirements(requests, limits api.ResourceList) api.ResourceRequ
 	return res
 }
 
-func validPod(name string, numContainers int, resources api.ResourceRequirements) *api.Pod {
+func validPod(name string, numContainers int, resources api.ResourceRequirements, priorityClass string) *api.Pod {
```

Comment:
this is a style thing, but looking back would have preferred we could have chained this call.

`validPod(foo, 3).WithResources(xyz).WithPriority(foo)`

Reply:
or `validPod(foo, 3, resources).WithPriority(foo)`